dnf-behave-tests/features/dnf-transaction-exception.feature

```diff
@@ -0,0 +1,15 @@
+@no_installroot
+Feature: Test exception when dnf tries to do a transaction resolved but not valid anymore
+
+Background: setup repository
+  Given I use repository "dnf-transaction"
+    And I configure dnf with
+        | key          | value                                 |
+        | pluginpath   | /dnf-plugins |
+    And I do not disable plugins
+
+
+Scenario: Dnf fails to update a plugin if it does not exist after resolving the transaction
+  Given I successfully execute dnf with args "install wilee"
+   When I execute dnf with args "remove update wilee"
+   Then the exit code is 0
```

Comment:
As I understand the problem you are trying to reproduce I would add to `dnf-transaction` repository another package let's say `llama-1.0.0`.
Than I would change `wilee` plugin to call `rpm -e llama` instead.
And the Scenario would be:
```
Given I successfully execute dnf with args "install llama"
And I successfully execute dnf with args "install wilee"
When I execute dnf with args "remove llama"
Then the exit code is 0
```
Current of dnf should raise an exception "_rpm.error: package not installed"
Does it make sense?

Reply:
it makes sense I am trying some changes now. Thank you